orbit/cmd/orbit/orbit.go

```diff
@@ -266,8 +271,6 @@ func main() {
 				// alarms when users look into the orbit logs, it's perfectly normal to
 				// not have a configuration profile, or to get into this situation in
 				// operating systems that don't have profile support.
-				case errors.Is(err, profiles.ErrNotImplemented), errors.Is(err, profiles.ErrNotFound):
-					log.Debug().Msgf("reading configuration profile: %v", err)
```

Comment:
Not sure I understand why adding the new flag makes that unnecessary?

Reply:
Ah I think I see, the `ErrNotFound` is not returned anymore when secret+ url are empty, but that doesn't matter because anyway we check if any of those are empty don't use them if that's the case.